lib/mvos/ui/widgets/drawer.dart

```diff
@@ -28,6 +29,21 @@ class CustomDrawer extends StatelessWidget {
               },
             ),
           ),
+          Divider(color: Colors.pink,thickness: 2,),
```

Comment:
All this should be one PlagroundButton widget. And then you make
```
kAppEnvironment == AppEnvironment.develop
                    ? PlaygroundButton(): Container();
```

Reply:
Znaci cijeli ovaj dio treba biti widget. I on treba ici ako je production/develop